crates/cubecl-cpu/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
We can create a simlink to the root licenses

Reply:
Done